client.cfg

```diff
@@ -28,12 +28,23 @@ ssh_tunnel=
 encounter_mapping=1
 patient_mapping=1
 
+[retcode]
```

Comment:
These return codes allow Jenkins to properly display the results of a Luigi run.  Without them most failed runs will appear successful.  

I accidentally deleted them while cleaning up other configuration parameters not needed for CDM.